### ISSUE TITLE
Why tilde requirements?

### ISSUE THREAD

**Issue by kuviman** [spacecowboy/html2runes#4]
Trying to use `structopt` and this crate in same project result in an error: [issue in structopt](https://github.com/TeXitoi/structopt/issues/139)

Since cargo can't have multiple semver-compatible versions of the same crate, it would fail if `structopt` changes its `clap` requirement to `>= version` where `version` is higher than version of `clap` in `html2runes`

Everything should have worked fine if `html2runes` had default (caret) requirements instead

**Comment by LukasKalbertodt** [comment]
I hit this bug, too.

Thanks for fixing this [in your fork](https://github.com/kuviman/html2runes), @kuviman. Could you send this as PR to this repo? It would be great if this could be merged. For future visitors: until this is fixed upstream, you can add this to your `Cargo.toml`:

```toml
[replace]
"html2runes:1.0.0" = { git = "https://github.com/kuviman/html2runes" }
```

At least as long as @kuviman "maintains" that fork.

**Comment by mdtusz** [comment]
This becomes an issue when using webgl with stdweb as well. @spacecowboy any channce this can be updated?

**Comment by spacecowboy** [comment]
Thanks for the ping @LukasKalbertodt . I have to be honest. I had no idea that i had any issues or pull requests on this repo.

I created a PR and merged @kuviman 's changes in #5 .

Will have a look and see if any of the deps should be updated as well.

As for why the requirements were listed with tilde, it's because of semver. Seems to be that `clap` must have broken semver by updating the API in a patch version?

**Comment by LukasKalbertodt** [comment]
Thanks for merging! :)
